### PR TITLE
bt-53: Create sidebar

### LINKED ISSUE #53: Create Sidebar
Image

Design: https://www.figma.com/file/Yd7USgP7PLvA0W794jHXWu/BSA_recruting?type=design&node-id=0-1&mode=design&t=lwbtBxp8JzQcOuO5-0

### PR DESCRIPTION
**Pull request by vladjsp**
sidebar2

### CODE REVIEW

**Comment by JemiJe** [comment]
I saw prev commit, but maybe we should leave ability to pass `menuItems` via props, but leave default items and add passed using, for example `menuItems = menuItems.concat(extraItems)`. As I see it, better to make component more extensible.

**Review by savinaDaria — APPROVED** [review]

File: frontend/src/bundles/common/components/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        link: AppRoute.SIGN_UP,
+        name: 'Chats',
+        icon: <EmailRounded />,
+    },
+];
+
+const Sidebar: React.FC = () => {
+    return (
+        <div className={styles.wrapper}>
+            <div className={styles.logo}></div>
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/sidebar/sidebar.tsx]: Leave some comment, that we should update it after we will have our logo

File: frontend/src/bundles/common/components/sidebar/sidebar.tsx
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/sidebar/sidebar.tsx:1]: you should import it to the root components.ts file

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
In general looks good

**Comment by Simpsonkk** [comment]
Probably we can also use z-index: 2 in .wrapper and position: fixed because the sidebar is always located above the header component.

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**